migrations/20200615162631-add-assess-again-flag-in-solutions.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  async up(db) {
+    global.migrationMsg = "Add allowMultipleAssessemts flag in solutions";
+
+    await db.collection('solutions').updateMany({
```

Comment:
@aman-tunerlabs - Add submission number in index of submissions collection please via another migration, similar to observation submissions.